exec/java-exec/src/main/java/org/apache/drill/exec/store/pcap/PcapDrillTable.java

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
The JIRA link is also good. Once the code is into Drill, it will be hard for future users to locate the corresponding JIRA. But, if you place a comment into a file somewhere, then future developers can find the JIRA.